pubspec.yaml

```diff
@@ -201,6 +202,8 @@ flutter:
     - packages/shrine_images/35-0.jpg
     - packages/shrine_images/36-0.jpg
     - packages/shrine_images/37-0.jpg
+    - assets/demos/motion/avatar_logo.png
```

Comment:
Presumably these will go away once you can use the package

Reply:
Using the package now.